.github/build.yaml.gomplate

```diff
@@ -529,6 +529,7 @@
       COSIGN_EXPERIMENTAL: 1 # use keyless signing
       COSIGN_REPOSITORY: raccos/releases-{{{ $flavor }}}
       PUBLISH_ARGS: "--concurrency 1 --plugin luet-cosign"
+      LUET_INSTALL_FROM_COS_REPO: false
```

Comment:
This causes to install the latest version of luet? Am I getting it right?

Reply:
So when we use `make deps` we use the scritp `get_luet.sh` to obtain the latest luet from the mudler/luet repo directly and use that + our repo to bootstrap the luet install, so we end up with luet installed from our releases-green repo and the releases-green repo configured in the luet.cfg file.

That proved difficult to use with a newly created arch/flavor as the repos on the first run will be empty, so  I modified the script to be able to just use the luet we download and copy that into the system with no repos configured[0]. So instead of downloading luet and installing luet from our cos repo, this other path will download luet and copy luet into the running system directly. This also bypasses whatever the version is on our repo and installs the latest (or you can override it with `LUET_VERSION` I think)


[0] https://github.com/rancher-sandbox/cOS-toolkit/commit/d4a2502af6d419f1b3ed9e25b252abe15dbf8fbd#diff-362156cbd5c315eb0e08dad196ecf6f0cf215fd71066c276ee46e0a624f5f647R17